src/sql/workbench/services/notebook/browser/notebookService.ts

```diff
@@ -199,11 +200,13 @@ export class NotebookRange extends Range {
 	}
 	cell: ICellModel;
 	isMarkdownSourceCell: boolean;
+	isCodeOutput: boolean;
```

Comment:
If this is going to be generic across different types of cells and whatnot then consider just having a single sourceType or something to cut down on the amount of `is...` vars you have to add (you already have two now with this new one)